src/lib/radio/radio.md

```diff
@@ -34,3 +34,13 @@ This internal radio button receives focus and is automatically labelled by the t
 `<mat-radio-button>` element.
 
 Radio button groups should be given a meaningful label via `aria-label` or `aria-labelledby`.
+
+### Default Color Configuration
+Color can be overridden on the provider level by using the following below:
```

Comment:
You can reword it a bit better like this: "The default color for radio buttons can be configured globally using the `MAT_RADIO_DEFAULT_OPTIONS` provider"

Reply:
Updated - also added valid color in example instead of `color-override`